packages/appeals-service-api/src/spec/appeal-case.yaml

```diff
@@ -493,6 +497,10 @@ components:
           description: the Inspector's outcome/decision for this case
         caseDecisionPublished:
           type: boolean
+        appealStatus:
+          type: string
+          enum: ['assign_case_officer', 'validation', 'ready_to_start', 'lpa_questionnaire_due', 'statement_review', 'final_comment_review', 'issue_determination', 'complete', 'invalid', 'withdrawn', 'closed', 'awaiting_transfer', 'transferred']
+          description: the current status of the appeal
```

Comment:
got this list of possible statuses from back office repo here: 
https://github.com/Planning-Inspectorate/appeals-back-office/blob/main/appeals/api/src/server/endpoints/constants.js

Reply:
looks like it matches the data model with some extras https://github.com/Planning-Inspectorate/data-model/blob/42bada8587fd5963f3ebb9e0269b2aecc19c2092/schemas/appeal-has.schema.json#L93